.sentry.example.yml

```diff
@@ -32,3 +32,8 @@ run_args:
 watch:
   - ./src/**/*.cr
   - ./src/**/*.ecr
+
+# The list of patterns of files to ignore from the watched file paths.
+ignore:
+  - /\.swp$/
+  - /^\.#/
```

Comment:
I'm sorry, what does the `#` do? I'm looking at the [pattern reference](https://crystal-lang.org/api/master/File.html#match%3F%28pattern%3AString%2Cpath%3AString%29-class-method) and don't see a hash character. You either meant `*` or I got some learning to do.

Reply:
Since these are strings that we have to cast to regex, does that mean we have to add extra backslashes to everything (e.g. `\\w`, `\\/`)?

Reply:
That example was inspired by the  issue I was having with the emacs lock files.  Those lock files take on the format, `.#my_file.cr`; So, that pattern filters them out (any file starting with `.#`).  The example patterns here could be better I think 🤔.  

Since we're reading in the `sentry.yml` with `File.read`, the escape backslashes are added automatically: `\nignore:\n  - /\\.swp$/\n  - /^\\.#/\n"`.